implementations/rust/ockam/ockam_api/src/kafka/portal_worker.rs

```diff
@@ -3,16 +3,17 @@ use core::sync::atomic::Ordering;
 use ockam_core::compat::sync::Arc;
 use ockam_core::{
     errcode::{Kind, Origin},
-    Address, AllowAll, Encodable, Error, LocalInfo, LocalMessage, Route, Routed, TransportMessage,
-    Worker,
+    Address, AllowAll, AsyncTryClone, Encodable, Error, LocalInfo, LocalMessage, Route, Routed,
+    TransportMessage, Worker,
 };
 use ockam_node::Context;
 use ockam_transport_tcp::{PortalMessage, MAX_PAYLOAD_SIZE};
 
 use crate::kafka::decoder::KafkaDecoder;
 use crate::kafka::encoder::KafkaEncoder;
 use crate::kafka::inlet_map::KafkaInletMap;
-use crate::kafka::protocol_aware::ProtocolState;
+use crate::kafka::protocol_aware::{ProtocolState, TopicUuidMap};
+use crate::kafka::secure_channel_map::KafkaSecureChannelController;
```

Comment:
Just a note on the diagram below. I think it would be even better it we could trace 2 boxes showing where each `KafkaWorker` in the portal is actually placed, with their 'receiving' role labelled. 
Right now I find the comments a bit confusing since they mention an "instance" of the "KafkaPortalWorker" which is actually "two different workers".